cvd/debian/rules

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/make -f
+
+# Start build by executing:
+# $ debuild --no-tgz-check -us -uc
+
+# Uncomment this line out to make installation process more chatty.
+# Keep it on until we know there's no outstanding problems with installation.
+# export DH_VERBOSE=1
+
+# There's a bug here
+export DEB_BUILD_MAINT_OPTIONS=hardening=-format
```

Comment:
Sorry I don't understand, what is the bug?

Reply:
I don't either, I just copied this file from our other packages.